src/icons/data.json

```
@@ -0,0 +1,268 @@
+      "filename": "location.svg"
+    },
+    "mail": {
+      "name": "Mail",
+      "url": "https://figma.com/file/R9KP76KdmjlWdKLZD9qTiw/Assets (Copy)?node-id=64:196",
+      "id": "mail",
+      "tags": [
+        "envelope"
+      ],
+      "svg": "<svg viewBox=\"0 0 20 20\"><path d=\"M18.75 2H1.25A1.25 1.25 0 000 3.25v13.5A1.25 1.25 0 001.25 18h17.5A1.25 1.25 0 0020 16.75V3.25A1.25 1.25 0 0018.75 2zm-1.29 2.5L10 10.405 2.54 4.5h14.92zM2.5 15.5V7.66L9.225 13a1.245 1.245 0 001.55 0L17.5 7.66v7.84h-15z\"/></svg>",
+      "filename": "mail.svg"
+    },
+    "pencil": {
+      "name": "Pencil",
+      "url": "https://figma.com/file/R9KP76KdmjlWdKLZD9qTiw/Assets (Copy)?node-id=64:197",
+      "id": "pencil",
+      "tags": [
+        "edit",
+        "write"
+      ],
+      "svg": "<svg viewBox=\"0 0 20 20\"><path d=\"M19.636 3.65L16.35.365a1.25 1.25 0 00-1.765 0l-13.22 13.22a1.21 1.21 0 00-.33.6l-1 4.28a1.245 1.245 0 001.5 1.5l4.28-1a1.21 1.21 0 00.6-.33l13.22-13.22a1.25 1.25 0 000-1.765zM15.466 3l1.5 1.5-1.215 1.265-1.5-1.5L15.466 3zM4.9 16.615l-2 .46.46-1.975L12.466 6l1.5 1.5L4.9 16.615z\"/></svg>",
+      "filename": "pencil.svg"
+    },
+    "phone": {
+      "name": "Phone",
+      "url": "https://figma.com/file/R9KP76KdmjlWdKLZD9qTiw/Assets (Copy)?node-id=64:198",
+      "id": "phone",
+      "tags": [
+        "telephone"
+      ],
+      "svg": "<svg viewBox=\"0 0 20 20\"><path d=\"M12.65 19.92a5.84 5.84 0 01-4.18-1.695l-6.57-6.57c-2.4-2.38-2.235-6.4.325-8.96L4.5.445a1.25 1.25 0 011.77 0l4.23 4.24a1.25 1.25 0 010 1.77L8.54 8.4l3.185 3.18 1.945-1.945a1.25 1.25 0 011.765 0l4.245 4.245a1.25 1.25 0 010 1.765L17.425 17.9a6.78 6.78 0 01-4.775 2.02zM4 4.465c-1.585 1.585-1.73 4-.325 5.42l6.56 6.57c1.405 1.405 3.84 1.26 5.425-.325l1.365-1.365-2.47-2.5-1.945 1.94a1.25 1.25 0 01-1.77 0l-4.95-4.95a1.245 1.245 0 010-1.765l1.945-1.945-2.475-2.45L4 4.465z\"/></svg>",
+      "filename": "phone.svg"
+    },
+    "search": {
+      "name": "Search",
+      "url": "https://figma.com/file/R9KP76KdmjlWdKLZD9qTiw/Assets (Copy)?node-id=64:199",
+      "id": "search",
+      "tags": [
+        "lens"
+      ],
+      "svg": "<svg viewBox=\"0 0 20 20\"><path d=\"M19.385 17.615l-5.78-5.775a7.5 7.5 0 10-1.765 1.765l5.775 5.78a1.255 1.255 0 001.77 0 1.26 1.26 0 000-1.77zM2.5 7.5a5 5 0 1110 0 5 5 0 01-10 0z\"/></svg>",
+      "filename": "search.svg"
+    },
+    "calendar": {
+      "name": "Calendar",
+      "url": "https://figma.com/file/R9KP76KdmjlWdKLZD9qTiw/Assets (Copy)?node-id=351:95",
+      "id": "calendar",
+      "tags": [
+        "date"
+      ],
+      "svg": "<svg viewBox=\"0 0 20 20\"><path d=\"M20 9V4a1.244 1.244 0 00-1.25-1.25H16v-1.5a1.25 1.25 0 00-2.5 0v1.5h-7v-1.5a1.25 1.25 0 00-2.5 0v1.5H1.25a1.275 1.275 0 00-.885.365A1.275 1.275 0 000 4v14.5c.002.331.133.649.365.885.236.232.554.363.885.365h17.5A1.244 1.244 0 0020 18.5V9zm-2.5-3.75v2.5h-15v-2.5h15zm-15 5h15v7h-15v-7z\"/></svg>",
+      "filename": "calendar.svg"
+    },
```

if we wanna go ahead with the hamburger menu:
```suggestion
    },
    "menu": {
      "name": "Menu",
      "url": "https://www.figma.com/file/P0usvPQSAKOzqcH5mPFf1u/Assets?node-id=1519%3A27",
      "id": "menu",
      "tags": [
        "hamburger"
      ],
      "svg": "<svg viewBox=\"0 0 20 20\"><rect x=\"2\" y=\"14\" width=\"16\" height=\"2\" rx=\"1\" fill=\"#002B48\"/><rect x=\"2\" y=\"9\" width=\"16\" height=\"2\" rx=\"1\" fill=\"#002B48\"/><rect x=\"2\" y=\"4\" width=\"16\" height=\"2\" rx=\"1\" fill=\"#002B48\"/></svg>",
      "filename": "menu.svg"
    },
```